.github/workflows/build-windows.yml

```diff
@@ -128,7 +128,7 @@ jobs:
             lfs: true
 
         # Install Flutter
-        - uses: subosito/flutter-action@v2
+        - uses: subosito/flutter-action@v2.12.0
```

Comment:
It's necessary to explicitly use v2.12.0 of flutter-action here since v2.13.0 is broken: https://github.com/subosito/flutter-action/issues/277#issuecomment-1974628019

Reply:
Thanks for letting me know.